docs/ref/contrib/admin/index.txt

```diff
@@ -2682,6 +2682,30 @@ app or per model. The following can:
 * ``search_form.html``
 * ``submit_line.html``
 
+``actions.html``
+====================
+
+.. attribute:: cl
```

Comment:
`.. attribute` must be assigned to something. To describe context we could use a list, e.g.
```
* ``actions.html`` with the following variables available in the template
  context:

  * ``cl``: ...
```